core/commands/swarm.go

```diff
@@ -61,6 +62,148 @@ const (
 	swarmDirectionOptionName = "direction"
 )
 
+type peeringResult struct {
+	ID     peer.ID
+	Status string
+}
+
+var swarmPeeringCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Modify the peering subsystem.",
+		ShortDescription: `
+'ipfs swarm peering' manages the peering subsystem. 
+Peers in the peering subsystem is maintained to be connected, reconnected 
+on disconnect with a back-off.
+`,
```

Comment:
Can we add a short note about changes not being saved?

Reply:
Sure, how about it?
```suggestion
The changes are not saved to the config.
`,
```

Reply:
SGTM!

Reply:
Thank you! I've taken it.